src/server/webpack-config.js

```diff
@@ -95,14 +103,15 @@ export default (options = {}) => {
       loaders: [
         babelLoader(/\.js$/, isRelayEnabled),
         babelLoader(/\.jsx$/, isRelayEnabled),
+        typescriptLoader(/\.tsx?$/, isRelayEnabled),
         { test: /\.json$/, loader: 'json-loader' },
         { test: /\.(png|svg)$/, loader: 'url-loader' },
       ],
     },
     devtool: isProduction ? undefined : 'cheap-module-eval-source-map',
     resolve: {
       moduleDirectories: NODE_MODULES_PATH,
-      extensions: ['', '.js', '.jsx', '.json'],
+      extensions: ['', '.js', '.jsx', '.json', '.ts', '.tsx'],
```

Comment:
Yes and no. `.ts` and `.tsx` are both typescript files, but JSX is only supported in `.tsx` files, not `.ts` files.